src/libtomahawk/Config.h.in

```diff
@@ -15,16 +15,18 @@
 
 #cmakedefine WITH_CRASHREPORTER
 #cmakedefine WITH_BINARY_ATTICA
-#cmakedefine WITH_QtSparkle
 #cmakedefine WITH_UPOWER
 #cmakedefine WITH_GNOMESHORTCUTHANDLER
 
 #cmakedefine LIBLASTFM_FOUND
 #cmakedefine QCA2_FOUND
+#cmakedefine QtSparkle_FOUND
```

Comment:
then cmake should fail (REQUIRED) when qtsparkle is not found. But it just fails during compile.